### ISSUE TITLE
Are you tax deductible? And more Information missing

### ISSUE THREAD

**Issue by rugk** [Exodus-Privacy/website#189]
I saw analysis is currently disabled? (It does not say why btw, do you need money for server costs?)
Point 1: if so, maybe mention that and link it there, there is no better advertising than to catch users in the process and say "yeah, we would like to do what you wanted to do us, but we're running out of money. <Help us here!>".

If so, I thought I check out the donation page.
Found it: https://exodus-privacy.eu.org/en/page/contribute/#finance

Point 2: That mail thing okay, but I clicked on a link on how to donate money and now it says "The membership fee remains free."?
Point 3: Also "Membership is to be renewed every year" aka I need to sent that mail each year, really? Also, if it is free, what is my or your benefit of me becoming a member actually? Sorry don't get it.

Point 4: what's striking and missing is what organisation (in legal terms) this goes ro and whether this is tax-deductable?
[Somewhere random on the website](https://exodus-privacy.eu.org/en/page/who/) I could find it is a "French (1901 law) non-profit organization". But what does that mean (I'm not from French)? Is my donation tax-deductable? (At least in French, that would be requirement no 1 to also be tax-deductable in other countries as Germany.)
Also in Germany you usually have some small PDF or so as a "proof" from your tax agency that you are a valid organisation and donations are tax-deductable or so. That would be nice.

Point 5: maybe you could also offer SEPA payments/an IBAN? In Europe that is a quite practical and cheap way to donate.

**Comment by Jean-BaptisteC** [comment]
Analysis is disabled because downloading apps from Play Store doesn't work. We are working on a fix.

**Comment by codeurimpulsif** [comment]
Hi @rugk and thanks for your issue. Yes analysis are disable because we encounter technicals issues which are not related to our finance.

- Point 2: Yes our membership fee remains free because we don't need this to have money, we run smoothly with only donations.
- Point 3: Yes members of our organization have to send an email each year to renew their membership. It's mostly to know if we need to keep someone in our "system" (mailing lists, git repositories, internal tools, etc) and to avoid old and unused accounts in our tools. Also, we like to know people who are part of our organization, it's important for us to keep human contact with members and be sure they are ok and still want to be members of our project. When someone become a member it's not to give us money but mostly to give us other things: ideas, knowledge, time, etc. Members are the persons who litteraly "run" Exodus Privacy, they participate to debates, decisions, interviews with journalists, resources creations, etc.
- Point 4: Yes we are a French non-profit according to the 1901 french law, which make us tax-deductable __in France__. But I don't known for other country, so I ping here our treasurer @imrhiel who maybe know the answer.
- Point 5: I remember we debate this some month ago during a meeting but we don't want to do so at the moment.

**Comment by imrhiel** [comment]
Hi @rugk,
In France, donations to us (Exodus Privacy association) are tax-deductible but we don't have a document to prove it because it is not mandatory to have this document if we fit the criteria, and seeing the small volume of donations where people ask us for a receipt for tax deduction, it doesn't really justify the paperwork to get this.
I looked into the question about being tax-deductible in Germany and from what I gathered (I'm not an expert at tax law in general and even less German tax law), while donations to foreign nonprofits can be deductible and the criteria seem pretty close to the French ones, I'm not entirely certain we fit everything nor that the documents we can provide you with (and which would need to be translated into German at your expense) would be enough to justify it to a German tax administration, seeing we are a small association entirely run by volunteers and I'm not a professional accountant. So I don't think it could be done for now, sorry.
